QUALITY_DECLARATION.md

```diff
@@ -0,0 +1,117 @@
+This document is a declaration of software quality for the `console_bridge_vendor` package, based on the guidelines in [REP-2004](https://www.ros.org/reps/rep-2004.html).
+
+# `console_bridge_vendor` Quality Declaration
+
+The package `console_bridge_vendor` claims to be in the **Quality Level 4** category.
+
+Below are the rationales, notes, and caveats for this claim, organized by each requirement listed in the [Package Quality Categories in REP-2004](https://index.ros.org/doc/ros2/Contributing/Developer-Guide/#package-quality-categories) of the ROS2 developer guide.
+
+## Version Policy [1]
+
+### Version Scheme [1.i]
+
+`console_bridge_vendor` uses `semver` according to the recommendation for ROS Core packages in the [ROS 2 Developer Guide](https://index.ros.org/doc/ros2/Contributing/Developer-Guide/#versioning).
+
+### Version Stability [1.ii]
+
+`console_bridge_vendor` is at or above a stable version, i.e. `>= 1.0.0`.
+
+### Public API Declaration [1.iii]
+
+This is a vendor package for `libconsole-bridge-dev` and as such does not declare its own API.
+
+### API Stability Policy [1.iv]
+
+`console_bridge_vendor` does not have an API of its own, and will limit its dependency on `libconsole-bridge-dev` to API stable releases within a released ROS distribution.
+
+### ABI Stability Policy [1.v]
+
+`console_bridge_vendor` does not have an ABI of its own, and will limit its dependency on `libconsole-bridge-dev` to ABI stable releases within a released ROS distribution.
+
+### ABI and ABI Stability Within a Released ROS Distribution [1.vi]
+
+As `console_bridge_vendor` does not come with its own API/ABI, changes will not affect API/ABI stability.
+
+## Change Control Process [2]
+
+`console_bridge_vendor` follows the recommended guidelines for ROS Core packages in the [ROS 2 Developer Guide](https://index.ros.org/doc/ros2/Contributing/Developer-Guide/#change-control-process).
+
+### Change Requests [2.i]
+
+All changes will occur through a pull request, check [ROS 2 Developer Guide](https://index.ros.org/doc/ros2/Contributing/Developer-Guide/#change-control-process) for additional information.
+
+### Contributor Origin [2.ii]
+
+This package uses DCO as its confirmation of contributor origin policy. More information can be found in [CONTRIBUTING](./CONTRIBUTING.md).
+
+### Peer Review Policy [2.iii]
+
+All pull requests will be peer-reviewed, check [ROS 2 Developer Guide](https://index.ros.org/doc/ros2/Contributing/Developer-Guide/#change-control-process) for additional information.
+
+### Continuous Integration [2.iv]
+
+This package is not being tested under ROS2 CI.
```

Comment:
It's part of `ros2.repos` though: https://github.com/ros2/ros2/blob/master/ros2.repos#L170

Reply:
The linters are not enabled on the `package.xml` file, I'll open another PR to fix this, already assigned myself in the spreadsheet.